edit/workload/index.vue

```diff
@@ -401,12 +403,20 @@ export default {
       this.$set(this.value, 'type', neu);
       delete this.value.apiVersion;
     },
+
+    errors(neu, old) {
+      // svc creation happened before workload, so if something went wrong with workload creation, svc should be deleted
+      if (neu && neu.length) {
+        this.cleanUpServices();
+      }
+    }
```

Comment:
Is there a reason to make the services before the workload?  There's lots of ways for the workload create to fail validation on the server-side so it would be cleaner if that was done first and there were no services to cleanup.